MWStoryGitFile.txt

```diff
@@ -27308,7 +27308,7 @@ Text inside of the PIPE signs (these things -> |text| ) are varibales, please do
 "Sun Sep 5 14:35:08 GMT+0200 2021_16770"MainText""
 "Sun Sep 5 14:35:08 GMT+0200 2021_16774"MainText"A device with much happy fluid inside for vaginal cleaning use."
 "Sun Sep 5 14:35:08 GMT+0200 2021_16778"MainText"This... ahm..."
-"Sun Sep 5 14:35:08 GMT+0200 2021_16778"If_story_text"You want to fill me up?"
+"Sun Sep 5 14:35:08 GMT+0200 2021_16778"If_story_text"You want to fill meup?"
```

Comment:
You reintroduced some spelling errors here which have been resolved by dubsington and the others already >_<

Reply:
I suspected that might happen, yes, unfortunately their changes conflicted with mine to a fairly significant extent, and I had roughly 2750 individual conflicts to resolve. I used my IDE's "resolve simple conflicts" function to process the bulk of them and it seems as though it didn't get it right every time. I'll go through and fix up the ones that you've spotted so far, there may be more to find.

Reply:
Should all be resolved now